pyface/message_dialog.py

```diff
@@ -30,6 +35,9 @@ def information(
         "Show details").
     informative : str
         Explanatory text to display along with the message.
+    text_format : str
+        Specifies what text format to use in the resulting message dialog.
+        One of "auto", "plain", or "rich".
```

Comment:
nit - it might be worth adding a short comment here - that this is only used by the Qt backend.